.github/workflows/moodle-plugin-ci.yml

```diff
@@ -0,0 +1,160 @@
+          - php: 7.3
+            moodle-branch: MOODLE_311_STABLE
+            database: mariadb
+          - php: 7.3
+            moodle-branch: MOODLE_310_STABLE
+            database: pgsql
+          - php: 7.3
+            moodle-branch: MOODLE_310_STABLE
+            database: mariadb
+          - php: 7.3
+            moodle-branch: MOODLE_39_STABLE
+            database: pgsql
+          - php: 7.3
+            moodle-branch: MOODLE_39_STABLE
+            database: mariadb
+
+          - php: 7.2
+            moodle-branch: MOODLE_310_STABLE
+            database: pgsql
+          - php: 7.2
+            moodle-branch: MOODLE_310_STABLE
+            database: mariadb
+          - php: 7.2
+            moodle-branch: MOODLE_39_STABLE
+            database: pgsql
+          - php: 7.2
+            moodle-branch: MOODLE_39_STABLE
+            database: mariadb
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          path: plugin
+
+      - name: Setup PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: ${{ matrix.extensions }}
+          ini-values: max_input_vars=5000
+          coverage: none
+
+      - name: Initialise moodle-plugin-ci
+        run: |
+          composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci ^3
+          echo $(cd ci/bin; pwd) >> $GITHUB_PATH
+          echo $(cd ci/vendor/bin; pwd) >> $GITHUB_PATH
+          sudo locale-gen en_AU.UTF-8
+          echo "NVM_DIR=$HOME/.nvm" >> $GITHUB_ENV
+
+      - name: Install moodle-plugin-ci
+        run: |
+          moodle-plugin-ci install --plugin ./plugin --db-host=127.0.0.1
+        env:
+          DB: ${{ matrix.database }}
+          MOODLE_BRANCH: ${{ matrix.moodle-branch }}
+          IGNORE_PATHS: moodle/tests/fixtures,moodle/Sniffs
+          CODECHECKER_IGNORE_PATHS: classes/vendor
+          PHPDOCCHECKER_IGNORE_PATHS: classes/vendor
+
+      - name: PHP Lint
+        if: ${{ always() }}
+        run: moodle-plugin-ci phplint
+
+      - name: PHP Copy/Paste Detector
+        continue-on-error: true # This step will show errors but will not fail
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpcpd
+
+      - name: PHP Mess Detector
+        continue-on-error: true # This step will show errors but will not fail
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpmd
+
+      - name: Moodle Code Checker
+        if: ${{ always() }}
+        run: moodle-plugin-ci codechecker --max-warnings 0 || true
+
+      - name: Moodle PHPDoc Checker
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpdoc || true
+
+      - name: Validating
+        if: ${{ always() }}
+        run: moodle-plugin-ci validate
+
+      - name: Check upgrade savepoints
+        if: ${{ always() }}
+        run: moodle-plugin-ci savepoints
+
+      - name: Mustache Lint
```

Comment:
this project do not have mustache, js, cs, unit and acceptance tests. Please comment out mustache, grunt, phpunit , behat steps, so it can be easily enabled, when they are realy needed

Reply:
I'll do!
Hang on.